internal/data/pg/daily_questions.go

```diff
@@ -0,0 +1,209 @@
+package pg
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/rarimo/geo-points-svc/internal/data"
+	"gitlab.com/distributed_lab/kit/pgdb"
+)
+
+const dailyQuestionsTable = "daily_questions"
+
+type dailyQuestionsQ struct {
+	db           *pgdb.DB
+	selector     squirrel.SelectBuilder
+	updater      squirrel.UpdateBuilder
+	counter      squirrel.SelectBuilder
+	deleter      squirrel.DeleteBuilder
+	timeSelector squirrel.SelectBuilder
+}
+
+func NewDailyQuestionsQ(db *pgdb.DB) data.DailyQuestionsQ {
+	return &dailyQuestionsQ{
+		db:           db,
+		selector:     squirrel.Select("*").From(dailyQuestionsTable),
+		updater:      squirrel.Update(dailyQuestionsTable),
+		deleter:      squirrel.Delete(dailyQuestionsTable),
+		counter:      squirrel.Select("COUNT(*) as count").From(dailyQuestionsTable),
+		timeSelector: squirrel.Select("*").From(dailyQuestionsTable).OrderBy("starts_at ASC"),
```

Comment:
You don't need this because pages have method apply in which you specified field for sort by